release.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python2.7
+
+import semver
+import xml.etree.ElementTree as ET
+import sys 
+
+"""Type: Major/Minor/Patch"""
+
+"""
+1. Read in values.xml
+2. Increment the SDK version value based on correct semVers
+3. Return: new version number
```

Comment:
This is really what `main` does, so should go under that function for clarity.

Reply:
done